src/cowboy_swagger.erl

```diff
@@ -11,6 +11,8 @@
 -export([filter_cowboy_swagger_handler/1]).
 -export([get_existing_definitions/1]).
 
+-hank([{unnecessary_function_arguments, [is_visible/2]}]).
```

Comment:
A comment above this line explaining why it's needed would be a nice addition.

Reply:
Sure. I'll do it later.

Reply:
Done

```erlang
% is_visible is used as a maps:filter/2 predicate, which requires a /2 arity function
```